### ISSUE TITLE
test-pattern.py example failing

### ISSUE THREAD

**Issue by tomrdyson** [pimoroni/drv2605-python#2]
When trying to run the `test-pattern.py` example I'm getting the following error:

```
python3 test-pattern.py --calibrate
test-pattern.py - Test a DRV2605 built-in haptic pattern

Calibrating...
Pattern: 1
Repeat: 1 time(s)
Delay: 100ms
Traceback (most recent call last):
  File "test-pattern.py", line 60, in <module>
    *sequence
  File "/usr/local/lib/python3.5/dist-packages/drv2605/__init__.py", line 300, in set_sequence
    self._drv2605.set('WAVEFORM_SEQUENCER', **settings)
  File "/usr/local/lib/python3.5/dist-packages/i2cdevice/__init__.py", line 219, in set
    self.set_field(register, field, value)
  File "/usr/local/lib/python3.5/dist-packages/i2cdevice/__init__.py", line 253, in set_field
    field = register.fields[field]
KeyError: 'set_step1_waveform'
```

The `test-waveform.py` works fine.

**Comment by Gadgetoid** [comment]
Whoops, I made a bit of a hash of that when I switched over to the new i2cdevice API. Working on a fix. Thank you!

**Comment by Gadgetoid** [comment]
Should be fixed as of 0.0.3!
